docs/reference/installation.rst

```diff
@@ -121,6 +121,12 @@ SonataPageBundle Configuration
             not_found: [404] # render 404 page with "not_found" key (name generated: _page_internal_error_{key})
             fatal: [500] # so you can use the same page for different http errors or specify specific page for each error
 
+        # This configuration is optional,
+        # If you don't want to load the sonata page's layout for your website just remove it, and load your css.
+        assets:
+            stylesheets:
+                - bundles/sonatapage/frontend.css
```

Comment:
IMO this does not look correct. https://github.com/sonata-project/SonataPageBundle/blob/4.x/src/DependencyInjection/Configuration.php#L140

This should be changed.

Reply:
ohhhh ok then I should create a app.css and let this as default

Reply:
No, frontend.css is fine, (there is already an app.css), but you need to change the configuration.

Reply:
Hey @jordisala1991 But we have a problem here, this app.js is used for admin not for FE (site)

Reply:
or should add this as default value? like

```php
                         ->defaultValue([
                                'bundles/sonatapage/app.css',
                                'bundles/sonatapage/frontend.css',
                            ])
```

Reply:
@jordisala1991  I have a question here, the page loads app.css and frontend.css, I guess we should load only frontend.css, don't you?

Reply:
the question is how!

Reply:
Ok we can handle this in other topic, let's make this working as 3.x

Reply:
you should only let the frontend.css line, the other remove it.

Reply:
Here is the problem

https://github.com/sonata-project/SonataPageBundle/blob/4.x/src/Resources/views/base_layout.html.twig#L25

it's loading all css, maybe a other solution could be add a new option like `frontend_ stylesheet`

what do you think?

Reply:
I would say it can be handled in other PR, because the current approach for this is loading app.css and it was loading bootstrap and admin css/js files in all pages, then we could provide some improvement later

Reply:
You should have to change configuration and only leave one file.